Formula/clisp.rb

```diff
@@ -30,27 +29,15 @@ class Clisp < Formula
     sha256               x86_64_linux: "46135189dd1acbfdcae88e81b78d90be6fce1d9b19e853d1c610cc9e146c8597"
   end
 
-  head do
-    url "https://gitlab.com/gnu-clisp/clisp.git"
-  end
-
   depends_on "libsigsegv"
   depends_on "readline"
 
   def install
-    ENV.deparallelize # This build isn't parallel safe.
-    ENV.O0 # Any optimization breaks the build
+    ENV.append "CC", "-m64"
```

Comment:
I think we have to use `ENV.m64` to add this flag, as the superenv will strip it out if it's in the compiler flags.  Do we know if this is actually needed?

Reply:
I don't think we have an `ENV.m64` method. We should probably stop filtering `-m64` now though, since we no longer support 32-bit machines.

Reply:
Homebrew/brew#12088

Reply:
This PR passes with `m64` being filtered, which means it’s not needed.  I would guess the need for it is a relic from the old version that is no longer relevant.

Reply:
that makes sense, I've removed it